docs/REST API/rest-api.md

```diff
@@ -3185,10 +3185,13 @@ This report includes a subset of host vitals, and simplified policy and vulnerab
     "updated_at": "2023-09-16T18:52:19Z",
     "os_version": "CentOS Linux 8.3.2011",
     "disk_encryption_enabled": true,
+    "failing_critical_policies_count": 1, // Fleet Premium only
```

Comment:
Is it potentially odd to have `failing_critical_policies_count` without also including the full `failing_policies_count`? (I realize you can just do .length on the array to get that info, but wondering if it’d be easier for folks using this endpoint for integrations to have it at the top-level.) lgtm either way, just wanted to bring that up!